Resources/Locale/en-US/accent/french.ftl

```diff
@@ -0,0 +1,125 @@
+accent-french-words-replace-17 = idiots
+accent-french-words-18 = do
+accent-french-words-replace-18 = faire
+accent-french-words-19 = viper
+accent-french-words-replace-19 = vipère
+accent-french-words-20 = officer
+accent-french-words-replace-20 = officier
+accent-french-words-21 = attention
+accent-french-words-replace-21 = l'attention
+accent-french-words-22 = op
+accent-french-words-replace-22 = opérationnel
+accent-french-words-23 = operative
+accent-french-words-replace-23 = opérationnel
+accentren-fch-words-24 = operatives
+accent-french-words-replace-24 = opérateurs
+accent-french-words-25 = coffee
+accent-french-words-replace-25 = café
+accent-french-words-26 = women
+accent-french-words-replace-26 = femmes
+accent-french-words-27 = thank you
+accent-french-words-replace-27 = merci beaucoup
+accent-french-words-28 = thanks
+accent-french-words-replace-28 = merci
+accent-french-words-29 = shuttle
+accent-french-words-replace-29 = navette
+accent-french-words-30 = enforcer
+accent-french-words-replace-30 = exécuteur
+accent-french-words-31 = ghost
+accent-french-words-replace-31 = fantôme
+accent-french-words-32 = weed
+accent-french-words-32-2 = cannabis
+accent-french-words-replace-32 = chanvre
+accent-french-words-33 = want
+accent-french-words-replace-33 = vouloir
+accent-french-words-34 = bomb
+accent-french-words-replace-34 = bombe
+accent-french-words-35 = shitcure
+accent-french-words-replace-35 = merdecure
+accent-french-words-36 = can
+accent-french-words-replace-36 = peux
+accent-french-words-37 = anomaly
+accent-french-words-replace-37 = anomalie
+accent-french-words-38 = artifact
+accent-french-words-38-2 = arti
+accent-french-words-replace-38 = artefact
+accent-french-words-39 = we
+accent-french-words-replace-39 = nous
+accent-french-words-40 = captain
+accent-french-words-replace-40 = capitaine
+accent-french-words-41 = awesome
+accent-french-words-replace-41 = parfaitement
+accent-french-words-42 = warning
+accent-french-words-replace-42 = avertissement
+accent-french-words-43 = warnings
+accent-french-words-replace-43 = avertissements
+accent-french-words-44 = and
+accent-french-words-replace-44 = et
+accent-french-words-45 = carp
+accent-french-words-replace-45 = carpe
+accent-french-words-46 = head
+accent-french-words-replace-46 = chapitre
+accent-french-words-47 = good
+accent-french-words-47-2 = nice
+accent-french-words-replace-47 = bon
+accent-french-words-48 = hi
+accent-french-words-replace-48 = salut
+accent-french-words-49 = hello
+accent-french-words-replace-49 = bonjour
+accent-french-words-50 = paramedic
+accent-french-words-replace-50 = paramédical
+accent-french-words-51 = paramed
+accent-french-words-replace-51 = paramédical
+accent-french-words-52 = bye
+accent-french-words-52-2 = goodbye
+accent-french-words-replace-52 = au revoir
+accent-french-words-53 = thing
+accent-french-words-replace-53 = chose
+accent-french-words-54 = things
+accent-french-words-replace-54 = choses
```

Comment:
this also will be hard to understand because `chose` is already a word in english